pkg/storaged/nfs-mounts.py

```diff
@@ -65,7 +66,8 @@ def index_tab(tab):
 
 
 def modify_tab(name, modify):
-    lines = open(name, "r").read().split("\n")
+    with open(name, "r") as f:
+        lines = f.read().split("\n")
     if len(lines) > 0 and lines[len(lines) - 1] == "":
```

Comment:
...`lines[-1]`...

Reply:
I mean, there's an awful lot that could be changed here, actually.  Perhaps it's out of scope for this PR.

Reply:
Right, but I at least did the simple `-1` change, that's obvious and safe.